features/step_definitions/admin_users_steps.rb

```diff
@@ -44,3 +50,32 @@
 Then(/^I expect the page to show me the number of admin users$/) do
   expect(page).to have_text("Admins (1)")
 end
+
+Then(/^I should see a column labeled "([^"]+)"$/) do |text|
+  find('th', text: text)
+end
+
+Given(/^there is a user in SAM.gov who is not a small business$/) do
+  @user = FactoryGirl.create(:user, sam_status: :sam_accepted, small_business: false)
+end
+
+Given(/^there is a user in SAM.gov who is a small business$/) do
+  @user = FactoryGirl.create(:user, sam_status: :sam_accepted, small_business: true)
+end
+
+Given(/^there is a user who is not in SAM.gov$/) do
+  @user = FactoryGirl.create(:user, sam_status: :sam_pending)
+end
+
+Then(/^I should see "([^"]+)" for the user in the "([^"]+)" column$/) do |value, column|
```

Comment:
do you think we should do that work in this PR, or wait until the other tests are being updated?